test/statetest/statetest_loader.cpp

```diff
@@ -11,6 +11,16 @@ namespace evmone::test
 namespace json = nlohmann;
 using evmc::from_hex;
 
+template <>
+uint8_t from_json<uint8_t>(const json::json& j)
+{
+    const auto ret = std::stoul(j.get<std::string>(), nullptr, 16);
+    if (ret > std::numeric_limits<uint8_t>::max())
+        throw std::out_of_range("from_json<uint8_t>: value > 0xFF");
```

Comment:
We may want to have a unit test for the exception case.

Reply:
Done